libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -109,6 +109,28 @@ const AP_Param::GroupInfo AP_ADSB::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("RF_SELECT",   9, AP_ADSB, out_state.cfg.rfSelect, UAVIONIX_ADSB_OUT_RF_SELECT_RX_ENABLED),
 
+    // @Param: SQUAWK
+    // @DisplayName: Squawk code
+    // @Description: VFR squawk (Mode 3/A) code is a pre-programmed default code when the pilot is flying VFR and not in contact with ATC. In the USA, the VFR squawk code is 1200 [0x04B0] and in most parts of Eurpope the VFR squawk code is 7000.
+    // @Unit: decimal
+    // @User: Advanced
+    AP_GROUPINFO("SQUAWK",  10, AP_ADSB, out_state.cfg.squawk, 1200),
+
+    // @Param: FLTPLAN_ID
+    // @DisplayName: Flightplan ID
+    // @Description: This is your 8 digit flightPlanID assigned from a filed flight plan. Set to 0 if you don't have a flightplan ID.
+    // @Range: 0 99999999
```

Comment:
This doesn't work out well, we only support parameter transfer via floats, and values greater then 2^24+1, and thus any value above 16,777,217 will be rounded off unexpectedly.